warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java

```diff
@@ -1,23 +1,23 @@
 package datawave.audit;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.util.List;
 
 import org.apache.commons.lang.math.IntRange;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import com.google.common.collect.Lists;
 
 import datawave.webservice.query.QueryImpl;
 
-public class SplitSelectorExtractorTest {
```

Comment:
curious why public was removed throughout.

Reply:
SonarLint will flag public modifiers in JUnit5 classes and test methods. Using the default package visible is recommended to improve readability.